app/views/layouts/_header.html.erb

```diff
@@ -43,15 +43,11 @@
         <li class="<%= 'active' if request.path =~ %r{^/deploys[/\?]?$} %>">
           <%= link_to "Deploys", deploys_path %>
         </li>
-        <li class="current-deploys <%= 'active' if current_page?(controller: '/deploys', action: 'active') %>" ng-controller="currentDeployBadgeCtrl">
+        <li class="<%= 'active' if current_page?(controller: '/deploys', action: 'active') %>">
```

Comment:
Would this be cleaner as logic in a ViewModel e.g. 
```
...
def initialize(active = false)
  @active = active
end

def display_active_class
  @active ? 'active' : ''
end
```

 or test for the presence of an instance variable instantiated in the controller instead?

Reply:
`@active` is not a global concept, any link can be active/inactive ... would be nice to do a `link_to_with_active` where it checks if the url is the current url or so ... but that's for another PR